test/inductor/test_torchinductor.py

```diff
@@ -5803,15 +5803,15 @@ def fn1(a, b):
             a = a.max(0).values
             c = torch.cat((a, b))
             c = c.round()
-            b >= a[0]
+            b >= a[0]  # noqa: B015
```

Comment:
huh do we intentionally not assert on these?

Reply:
I did not touch it because it's inductor and testing code compilation so adding an assert here would change what is being tested.

Reply:
Yea, based on the linked issue, this meaningless line looks crucial here haha.